README.md

```diff
@@ -91,31 +91,31 @@ purpose other than to help maintain clean repositories on behalf of the communit
 
 ### Create a RFC
 
-RFC (request for comments) are made using the Github issue tracker of the [php-cds/php-cds](https://github.com/php-cds/php-cds/issues) repository. You can create an issue, add a title prefixed with [RFC] and
+RFCs (request for comments) are made using the GitHub issue tracker of the [php-cds/php-cds](https://github.com/php-cds/php-cds/issues) repository. You can create an issue, add a title, prefixed with `[RFC]` and
 start writing about your idea. That's it.
 
 Try to be as concise as possible so readers will be able to quickly digest and understand your idea before voting.
 
 ## How to vote on an RFC
 
-In the Github issue tracker of the php-cds/php-cds repository you see all RFCs (issues with a title prefixed by `[RFC]`).
-After reading the issue description, try to understand it and express your opinion with a [Github reaction](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments)
+In the GitHub issue tracker of the [php-cds/php-cds](https://github.com/php-cds/php-cds) repository you see all RFCs (issues with a title prefixed with `[RFC]`).
+After reading the issue description, try to understand it and express your opinion with a [GitHub reaction](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments)
 using :+1: and :-1:.
 
 ## Implement/discuss an RFC
 
-Once you think you have enough positive feedback from the community (as seen by the [Github reactions](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments))
+Once you think you have enough positive feedback from the community (as seen by the [GitHub reactions](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments))
 you are free to open a discussion thread in the php-cds/php-cds-discussion repository. People that are interested in joining
-your RFC can discuss and reply at any time. You can also create Pull-Requests so people can easily comment on proposed code.
+your RFC can discuss and reply at any time. You can also create pull requests so people can easily comment on proposed code.
 
 ## Publish PSR
 
-Once on a RFC has positive community reaction, you have decided to create a discussion and a concrete implemention, and you think your work is
+Once an RFC has positive community reaction, you have decided to create a discussion and a concrete implemention, and you think your work is
```

Comment:
I'm not a native but I remembered the rule in the way that it's not based on the pronouncement but on the actual first letter. Anyone knows the actual truth™? :D